helm/charts/kratos/values.yaml

```diff
@@ -55,27 +58,106 @@ ingress:
     # kubernetes.io/tls-acme: "true"
     hosts:
       -
-        host: chart-example.local
-        paths: []
+        host: kratos.public.local.com
+        paths:
+          - path: /
+            backend:
+              servicePort: 80
 
     tls: []
     #  - secretName: chart-example-tls
     #    hosts:
     #      - chart-example.local
 
-hive:
+kratos:
   development: false
   autoMigrate: false
+
   config:
-    dsn: memory
-    secrets:
-      session: []
     serve:
       admin:
         port: 8081
       public:
         port: 8080
 
+    dsn: postgresql://postgres:123456@kratos-postgresql/kratos?sslmode=disable
+    selfservice:
+      strategies:
+        password:
+          enabled: true
+
+      verify:
+        return_to: http://127.0.0.1:4455/
+
+      logout:
+        redirect_to: http://127.0.0.1:4455/auth/login
+
+      login:
+        request_lifespan: 10m
+        after:
+          password:
+            -
+              job: session
+            -
+              job: redirect
+              config:
+                default_redirect_url: http://127.0.0.1:4455/
+                allow_user_defined_redirect: true
+
+      registration:
+        request_lifespan: 10m
+        after:
+          password:
+            -
+              job: verify
+            -
+              job: session
+            -
+              job: redirect
+              config:
+                default_redirect_url: http://127.0.0.1:4455/
+                allow_user_defined_redirect: true
+
+    log:
+      level: debug
+
+    secrets:
+      session:
+        - PLEASE-CHANGE-ME-I-AM-VERY-INSECURE
+
+    urls:
+      login_ui: http://127.0.0.1:4455/auth/login
+      registration_ui: http://127.0.0.1:4455/auth/registration
+      error_ui: http://127.0.0.1:4455/error
+      profile_ui: http://127.0.0.1:4455/profile
+      verify_ui: http://127.0.0.1:4455/verify
+
+    # These are undefined because not available in this demo
+      mfa_ui: http://127.0.0.1:4455/
+
+      self:
+        public: http://127.0.0.1:4455/.ory/kratos/public/
+        admin: http://kratos:4434/
+      default_return_to: http://127.0.0.1:4455/
+      whitelisted_return_to_domains:
+        - http://127.0.0.1:4455
+
+    hashers:
+      argon2:
+        parallelism: 1
+        memory: 131072
+        iterations: 2
+        salt_length: 16
+        key_length: 16
+
+    identity:
+      traits:
+        default_schema_url: file:///etc/config/kratos/identity.traits.schema.json
+
+    courier:
+      smtp:
+        connection_uri: smtps://test:test@mailslurper:1025/?skip_ssl_verify=true
+
```

Comment:
The should not be set - the configuration docs should be looked up in ORY Kratos. The default values will just cause confusion because some part of the system doesn't work :)

```suggestion
```